### PR TITLE
Removes illegal tech from syndicate thrown weapons

### LINKED ISSUE #47792: Metastation roundstart bar/armory throwing kit box has illegal tech
Syndicate boxes have illegal tech for some reason, it's likely a bad thing that illegal tech spawns roundstart on metastation

### PR DESCRIPTION
**Pull request by TheVekter**
Fixes #47792. Also added a single throwing star to maint loot per request because reasons.

## Why It's Good For The Game

Makes it harder to get illegal tech early on in the round. Should be a reward for dunking on a bad traitor, not making it to the bar first.

## Changelog
:cl:
tweak: Removed the ability to unlock illegal tech using the Syndicate thrown weapons (Paper plane, throwing star, bola)
add: Added a fun little surprise to maintenance loot!
/:cl:

### CODE REVIEW

**Comment by pireamaineach** [comment]
Improve don't remove



My right to RNG antag gamer gear shan't be infringed.

**Comment by TheVekter** [comment]
The antag stuff in maintenance is still there.

**Comment by Tlaltecuhtli** [comment]
can u add single  ninja stars in the uncommon maint loot spawns

**Comment by TheVekter** [comment]
> can u add single ninja stars in the uncommon maint loot spawns

Might do this.

**Comment by carlarctg** [comment]
free 20 damage weapon lol

lol

lol @ 6 downreddits, 3 tc antag gear for free roundstart is bad, imagine if janitor got noslips or something

File: code/modules/projectiles/guns/ballistic/revolver.dm [hunk 1]
```diff
@@ -138,6 +138,7 @@
 	name = "\improper Unica 6 auto-revolver"
 	desc = "A retro high-powered autorevolver typically used by officers of the New Russia military. Uses .357 ammo."
 	icon_state = "mateba"
+	var/illegal_tech = FALSE
```
Comment (ExcessiveUseOfCobblestone) [inline code/modules/projectiles/guns/ballistic/revolver.dm]: does this do anything?

**Comment by zxaber** [comment]
The ninja stars being there are almost iconic at this point.

Having a reasonable excuse for why a person might have ninja stars makes the game more fun.

File: code/modules/projectiles/guns/ballistic/revolver.dm (hunk 1 above)
Comment (TheVekter) [reply code/modules/projectiles/guns/ballistic/revolver.dm]: Yes, it removes the Mateba from the illegal tech list so when you deconstruct it you aren't getting illegal tech unlocked from it.

Comment (actioninja) [reply code/modules/projectiles/guns/ballistic/revolver.dm]: this doesn't make any sense, you're defining a new var and then never referencing it anywhere

Comment (TheVekter) [reply code/modules/projectiles/guns/ballistic/revolver.dm]: Let me mess with this a bit. To my knowledge it's defined in the research module as "if this is false it isn't worth illegal tech", but I'm going off of someone else's info on this.

Comment (TheVekter) [reply code/modules/projectiles/guns/ballistic/revolver.dm]: I'm doing this wrong. Ignore this, I'm changing it.

**Comment by TheVekter** [comment]
Alright, since folks wanted this, I shall deliver.

**Comment by wesoda25** [comment]
> The ninja stars being there are almost iconic at this point.
> 
> Having a reasonable excuse for why a person might have ninja stars makes the game more fun.

I think out of place is more appropriate than iconic. Big disagree on second part too, just results in more non antags running around with antag gear.

**Comment by Tharcoonvagh** [comment]
Ive got a feeling that this will completely remove these items from the uplink

**Comment by TheVekter** [comment]
image

Nope, already checked for that.

**Comment by carlarctg** [comment]
@zxaber CULTURE ARGUMENT :scream:

**Pull request merged**